<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formbuktipembayaran</class>
 <widget class="QWidget" name="Formbuktipembayaran">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>341</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nobuktiLabel">
      <property name="text">
       <string>nobukti</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nobuktiLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="noinvoiceLabel">
      <property name="text">
       <string>noinvoice</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="noinvoiceLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="tglbuktiLabel">
      <property name="text">
       <string>tglbukti</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateTimeEdit" name="tglbuktiDateTimeEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="keteranganLabel">
      <property name="text">
       <string>keterangan</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="keteranganLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="norekLabel">
      <property name="text">
       <string>norek</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="norekLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
